models/project.py

```diff
@@ -0,0 +1,64 @@
+from google.appengine.ext import ndb
+
+
+class Project(ndb.Model):
+
+    title = ndb.StringProperty()
+    member_keys = ndb.KeyProperty(repeated=True, kind='User')
+    task_keys = ndb.KeyProperty(repeated=True, kind='Task')
+    course = ndb.KeyProperty()
+
+    @property
+    def members(self):
+        return map(lambda k: k.get(), self.member_keys)
+
+    def tasks(self):
+        return map(lambda k: k.get(), self.task_keys)
+
+    @property
+    def url(self):
+        return '/project?key={}'.format(self.key.urlsafe())
+
+    @classmethod
+    def new(cls, title):
```

Comment:
We actually don't need this method implemented here.  It was necessary in the Course model because we needed to add to the DB a way to search for course by title, but that's not necessary for Projects.  We should just use the default constructor instead.

Additionally, the one place that this method was invoked calls it with two parameters, like this:

```python
project = Project.new(title, course_key)
```

even though the method signature only has one parameter (plus the class itself, which doesn't really count).